src/fusion_pipeline.py

```diff
@@ -24,10 +23,11 @@ class FusionPipeline:
         The fusers to apply, in order.
     """
 
-    def __init__(self, fusers):
+    def __init__(self, fusers=[], region_growing=[]): # TODO =None kan niet vanwege de for loop
```

Comment:
Is dit de juiste manier

Reply:
Of wel None als default en dan met een `if .. is not None` checken.